runtime/autoload/man.vim

```diff
@@ -112,7 +154,7 @@ endfunction
 
 function! s:get_path(sect, name) abort
   if empty(a:sect)
-    let path = system(s:man_cmd.' '.s:man_find_arg.' '.shellescape(a:name))
+    let path = s:system(['man', s:man_find_arg, a:name])
     if path !~# '^\/'
```

Comment:
We shouldn't need this anymore, s:system() will throw an exception because man's error status will be non zero if it does not find a manpage.